CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
 ## [Master] - 2025-03-21
 
+### Changed
+
+- MINOR Refactor the VOF interface sharpening and reinitialization parameters to merge them in one subsection called interface regularization method. [#1467](https://github.com/chaos-polymtl/lethe/pull/1467)
```

Comment:
I would consider every parameter change from now on a major change since it breaks PRM Files.